### PR TITLE
Adding containerd-node-throughput job config

### PR DESCRIPTION
**Pull request by piyushgupta1551**
Adding node-throughput prow job config 
Adding namespace metadata to perfdash deployment template.

### CODE REVIEW

File: config/jobs/periodic/kubernetes/test-kubernetes-periodics.yaml [hunk 1]
```diff
@@ -176,3 +176,76 @@ periodics:
...
+
+              TIMESTAMP=$(date +%s)
+              K8S_BUILD_VERSION=$(curl https://storage.googleapis.com/k8s-release-dev/ci/latest.txt)
+              jq --arg key0 'k8s-build-version' --arg value0 $K8S_BUILD_VERSION '. | .[$key0]=$value0' <<<'{}' > $ARTIFACTS/metadata.json
+
+              # kubectl needed for the e2e tests
+              curl -sSL https://dl.k8s.io/ci/$K8S_BUILD_VERSION/bin/linux/`go env GOARCH`/kubectl > /usr/local/bin/kubectl
+              chmod +x /usr/local/bin/kubectl
+
+              pushd /tmp;git clone https://github.com/kubernetes/perf-tests.git
```
Comment (mkumatag) [inline config/jobs/periodic/kubernetes/test-kubernetes-periodics.yaml:234]: can you put these changes into `extra_refs` section which take care of git cloning

File: config/jobs/periodic/kubernetes/test-kubernetes-periodics.yaml [hunk 2]
```diff
@@ -176,3 +176,76 @@ periodics:
...
+              popd
+
+              kubetest2 tf --powervs-dns k8s-tests \
+                --powervs-image-name centos-83-12142020 \
```
Comment (mkumatag) [inline config/jobs/periodic/kubernetes/test-kubernetes-periodics.yaml:238]: we already have an image in the tier1 with name `centos-83-03222021-tier1`, you can mentioned that here

File: config/prow/perfdash/perfdash-deployment.yaml [hunk 3]
```diff
@@ -2,6 +2,7 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: perfdash
+  namespace: perfdash
```
Comment (mkumatag) [inline config/prow/perfdash/perfdash-deployment.yaml:5]: It will be good if you can submit this change in a different PR

**Review by mkumatag — CHANGES_REQUESTED** [review]

**Comment by piyushgupta1551** [comment]
Closing this PR. Will raise 2 new ones with review comments as I need to delete my fork and redo the changes due to branch conflicts at my end.

**Pull request closed without merging**